src/commands/cont.rs

```diff
@@ -47,3 +66,61 @@ impl ContinueCommand {
         Ok(())
     }
 }
+
+fn get_time_to_continue(
+    time_entries: Vec<TimeEntry>,
+    running_time_entry: Option<TimeEntry>,
+    interactive: bool,
+) -> Option<TimeEntry> {
+    match interactive {
+        true => get_time_entry_from_user(time_entries),
+        false => get_first_stopped_time_entry(time_entries, running_time_entry),
+    }
+}
+
+fn get_first_stopped_time_entry(
+    time_entries: Vec<TimeEntry>,
+    running_time_entry: Option<TimeEntry>,
+) -> Option<TimeEntry> {
+    // Don't continue a running entry that was just stopped.
+    let continue_entry_index = match running_time_entry {
+        None => 0,
+        _ => 1,
+    };
+    return time_entries.get(continue_entry_index).cloned();
+}
+
+fn get_time_entry_from_user(time_entries: Vec<TimeEntry>) -> Option<TimeEntry> {
+    let options = SkimOptionsBuilder::default()
+        .height(Some("100%"))
+        .multi(false)
+        .build()
+        .unwrap();
+
+    let (sender, source): (SkimItemSender, SkimItemReceiver) = unbounded();
+    time_entries.iter().for_each(|time_entry| {
+        let _ = sender.send(Arc::new(time_entry.clone()));
+    });
```

Comment:
I really don't like the for_each method. A simple imperative for loop reads much better IMO

Reply:
✅